files/rb_task_helper.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module PuppetAgent::RbTaskHelper
+    private
+
+    def error_result(error_type, error_message)
+      {
+        '_error' => {
+          'msg' => error_message,
+          'kind' => error_type,
+          'details' => {},
+        },
+      }
+    end
+
+    def puppet_bin_present?
```

Comment:
I see this is pulled from the [run task](https://github.com/puppetlabs/puppetlabs-puppet_agent/blob/main/tasks/run.rb#L20) - any chance we can use this file as a helper there too and remove the function definitions from there? That way if these change at all we don't have to change it in two places, or end up with divergent implementations.